src/tb/tb_system_sdl.cpp

```diff
@@ -8,9 +8,17 @@
 
 #ifdef TB_SYSTEM_SDL
 
+#include "tb_msg.h"
+#include "tb_types.h"
 #include <sys/time.h>
 #include <stdio.h>
 
+#ifdef __EMSCRIPTEN__
```

Comment:
This should just be <SDL.h> and let pkg-config (resp. FindPackage) set up the cflags for the include search paths. A lot of linux distributions are handling this differently. You also don't need the emscripten hack anymore than.

pkg-config --cflags SDL2 should also give you /usr/include/SDL2 on linux.